README.md

```diff
@@ -171,6 +171,169 @@ If any functionality you need is not listed here, feel free to create a PR with
     * Error: `Failed`
 ---
 
+### Equalizer
+
+* #### Disable Equalizer
+  * Command: `EQOff`
+  * Args: None
+  * Output: `OK`
+
+* #### Get list of available Equalizer presets
```

Comment:
Please provide URL examples for
* `EQOff`
* `EQGetList`
* `Squeezelite:getState`
* `Squeezelite:discover`